src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1920,3 +1782,133 @@ def _process_privacy_request_restart(
     )
 
     return privacy_request  # type: ignore[return-value]
+
+
+@router.get(
+    REQUEST_TASKS,
+    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_READ])],
+    response_model=List[PrivacyRequestTaskSchema],
+)
+def get_individual_privacy_request_tasks(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+) -> List[RequestTask]:
+    """Returns individual Privacy Request Tasks created by DSR 3.0 scheduler
+    in order by creation and collection address"""
+    pr: PrivacyRequest = get_privacy_request_or_error(db, privacy_request_id)
+
+    logger.info(f"Getting Request Tasks for '{privacy_request_id}'")
+
+    return pr.request_tasks.order_by(
+        RequestTask.created_at.asc(), RequestTask.collection_address.asc()
+    ).all()
+
+
+@router.get(
+    PRIVACY_REQUEST_ACCESS_DATA,
+    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_VIEW_DATA])],
+    response_model=Dict,
+)
+def get_access_data(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+) -> Dict[str, Dict[str, List[Row]]]:
+    """Returns filtered data collected for an access request by policy rule key."""
+    pr = get_privacy_request_or_error(db, privacy_request_id)
+
+    if not pr.policy.get_rules_for_action(action_type=ActionType.access):
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail=f"Cannot retrieve access data for {pr.id}. This is not an access request.",
+        )
+
+    if pr.status != PrivacyRequestStatus.complete:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail=f"Cannot retrieve access data for incomplete request {pr.id}.",
+        )
+
+    logger.info(f"Getting privacy request '{privacy_request_id}' access data")
+    return pr.get_filtered_access_results()
```

Comment:
I am thinking we pull this endpoint out now that I am seeing revised requirements for https://ethyca.atlassian.net/wiki/spaces/PM/pages/3010101257/Self-service+access+package+downloads